spec/messages.js

```diff
@@ -2673,6 +2673,54 @@
             }));
         });
 
+        describe("Refuse bad mention pattern", function () {
+
+            it("Get parsed but shouldn't",
```

Comment:
You don't need to put this inside another `describe` call and the `it` call needs to go one level up because this test is not about received messages.

Also, please make the test name descriptive. The name must read correctly as if you're starting a sentence with `It`.

So for example 
```
describe("A Groupchat Message", function () { // This is already in the tests, don't add it
    it("parses for mentions as indicated with an @ preceded by a space or at the start of the text"`.
```